### PR TITLE
fix(solarcal): Using __file__ to load Solarcal splines from CSV

### PR DESCRIPTION
**Pull request by chriswmackey**
This is much cleaner.  Thank you for the suggestion @mostapharoudsari

### CODE REVIEW

File: ladybug_comfort/solarcal.py [hunk 1]
```diff
@@ -21,18 +21,19 @@
...
 
 def _load_solarcal_splines():
     """load the spline data that gets used in solarcal."""
     try:
-        cur_dir = dirname(abspath(inspect.getfile(inspect.currentframe())))
+        cur_dir = dirname(__file__)
         solarcal_splines = {
-            'seated': csv_to_num_matrix(cur_dir + '/_mannequin/seatedspline.csv'),
-            'standing': csv_to_num_matrix(cur_dir + '/_mannequin/standingspline.csv')}
+            'seated': csv_to_num_matrix(join(
```
Comment (mostaphaRoudsari) [inline ladybug_comfort/solarcal.py]:
```suggestion
            'seated': csv_to_num_matrix(os.path.join(
```

File: ladybug_comfort/solarcal.py [hunk 2]
```diff
@@ -21,18 +21,19 @@
...
+                cur_dir, '_mannequin', 'seatedspline.csv')),
+            'standing': csv_to_num_matrix(join(
```
Comment (mostaphaRoudsari) [inline ladybug_comfort/solarcal.py]:
```suggestion
            'standing': csv_to_num_matrix(os.path.join(
```

File: ladybug_comfort/solarcal.py [hunk 3]
```diff
@@ -21,18 +21,19 @@
 from ladybug.skymodel import calc_sky_temperature
 from ladybug.futil import csv_to_num_matrix
 
-from os.path import dirname, abspath
-import inspect
+from os.path import dirname, join
```
Comment (mostaphaRoudsari) [inline ladybug_comfort/solarcal.py]: not sure how I feel about overwriting the python `join`. I would import os and fully type the commands but up to you.

File: ladybug_comfort/solarcal.py [hunk 4]
```diff
@@ -21,18 +21,19 @@
...
 import math
 
 
 def _load_solarcal_splines():
     """load the spline data that gets used in solarcal."""
     try:
-        cur_dir = dirname(abspath(inspect.getfile(inspect.currentframe())))
+        cur_dir = dirname(__file__)
```
Comment (mostaphaRoudsari) [inline ladybug_comfort/solarcal.py]:
```suggestion
        cur_dir = os.path.dirname(__file__)
```

**Comment by chriswmackey** [comment]
Ok @mostaphaRoudsari .  I will change it to use the full `os.path` and then merge it in.

**Pull request merged**